prql-compiler/Cargo.toml

```diff
@@ -122,3 +122,4 @@ search = 'prefix-key: [\d.]+'
 
 [features]
 test-external-dbs = []
+test-website = []
```

Comment:
Why have this as an extra feature?

The external dbs requires more infra so needs the flag, but otherwise everything can run under `cargo test`...